src/components/SpcChart/SpcChart.tsx

```diff
@@ -176,6 +179,7 @@ export function SpcChart(props: Props) {
       gradientMode: GraphGradientMode.Opacity,
       lineWidth: lineWidth,
       lineInterpolation: LineInterpolation.Smooth,
+      drawStyle: drawStyle, //    Bars = "bars",    Line = "line",     Points = "points"
```

Comment:
This seems to be an unnecessary comment. Could you please remove it, or explain how it adds additional context to the code? 😆

Reply:
Right, I missed that 🙈